front/app/containers/Admin/formBuilder/components/FormBuilderToolbox/index.tsx

```diff
@@ -85,10 +97,20 @@ const FormBuilderToolbox = ({
             onClick={() => addAnswer('text')}
           />
           <ToolboxItem
-            icon="info"
+            icon="multiple-choice"
             label={formatMessage(messages.multipleChoice)}
             onClick={() => addAnswer('multiselect')}
           />
+          <ToolboxItem
+            icon="info"
```

Comment:
Please ask Ben for a more specific icon. Given that Ben is not working today, let's not block merging this PR due to this issue.